pages/index.vue

```diff
@@ -83,7 +83,7 @@ p {
 }
 
 .hero-image {
-  background-image: url('/olivia-bliss-HTnycx0h_kk-unsplash.jpg');
+  background-image: url('/hardik-pandya--Ey_0PMz900-unsplash.jpg');
```

Comment:
do you mind renaming this to something sane while you're here? e.g. `san-francisco-skyline.jpg`

Reply:
yup makes sense - done!